zebra/zebra_vty.c

```diff
@@ -915,8 +915,8 @@ static void do_show_route_helper(struct vty *vty, struct zebra_vrf *zvrf,
 	}
 }
 
-static void do_show_ip_route_all(struct vty *vty, struct zebra_vrf *zvrf, afi_t afi,
-				 bool use_fib, bool use_json,
+static void do_show_ip_route_all(struct vty *vty, struct zebra_vrf *zvrf,
```

Comment:
Why the whitespace changes here?

Reply:
this was >80; since I happened to be "here", I figured I'd re-align it